src/IssueViz.Security/Commands/HotspotsToolWindowCommand.cs

```diff
@@ -37,12 +36,13 @@ internal sealed class HotspotsToolWindowCommand
         
         public static HotspotsToolWindowCommand Instance { get; set; }
 
-        private readonly AsyncPackage package;
+        private readonly IToolWindowService toolWindowService;
```

Comment:
The package is now only used in the static initialization method to fetch the services and MEF components.